### PR TITLE
feat: [] adds `open-settings` command to app-scripts

### PR DESCRIPTION
**Pull request by loweisz**
Adds a simple open command to app-scripts. 
The structure of the call is a bit different as there is no sense to have a interactive/non-interactive mode, as this command will only run in one place and never in a CI. 

So there is only one function executed.

### CODE REVIEW

**Review by shikaan — COMMENTED** [review]
LGTM 🎉 

However, given that `--ci` is a top level flag, I would keep the interactive/non-interactive structure and throw if you run the command in non-interactive mode (as we do for create app definition). Feels more predictable

**Comment by loweisz** [comment]
> LGTM 🎉
> 
> However, given that `--ci` is a top level flag, I would keep the interactive/non-interactive structure and throw if you run the command in non-interactive mode (as we do for create app definition). Feels more predictable

@shikaan I also thought about that, but would you show the prompt to select the definition from the list in interactive mode ? 

Somehow the interactive mode didn't really felt useful to me just to open the settings, as you would need to get the access-token, orgId to have the list of definitions to select. 

Or we make the prompt just a text field to pass in the definition id instead of a list to select from, wdyt?

**Comment by shikaan** [comment]
> > LGTM 🎉
> > However, given that `--ci` is a top level flag, I would keep the interactive/non-interactive structure and throw if you run the command in non-interactive mode (as we do for create app definition). Feels more predictable
> 
> @shikaan I also thought about that, but would you show the prompt to select the definition from the list in interactive mode ?
> 
> Somehow the interactive mode didn't really felt useful to me just to open the settings, as you would need to get the access-token, orgId to have the list of definitions to select.
> 
> Or we make the prompt just a text field to pass in the definition id instead of a list to select from, wdyt?

@loweisz I agree 100% that it's not useful :) I was just speaking about the shape of the code. What I was trying to convey is that we should always keep the command structure predictable (therefore always taking in account the `--ci` flag), but when one of the behaviours - either the interactive or the non-interactive - does not make sense, we throw an exception.

The example I was referring to is this one
https://github.com/contentful/create-contentful-app/blob/7adcb0f3ca538143661be24623d80a9511617d6e/packages/contentful--app-scripts/lib/create-app-definition/index.js#L5-L15

Does it make sense to you?

File: packages/contentful--app-scripts/lib/open/open-settings.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    console.log(`
+        ${chalk.red('Error:')} There was no app-definition defined.
+
+        Please add it with ${chalk.cyan('--definition-id=<app-definition-id>')}
+        or set the environment variable ${chalk.cyan(`${APP_DEF_ENV_KEY} = <app-definition-id>`)}
+      `);
+    throw new Error('No app-definition-id');
+  }
+
+  open(`${REDIRECT_URL}&id=${definitionId}`);
```
Comment (kdamball) [inline packages/contentful--app-scripts/lib/open/open-settings.js:38]: I'd wrap this in a try just in case. We do this in getMgmtToken also

**Review by shikaan — APPROVED** [review]

File: packages/contentful--app-scripts/lib/open/open-settings.js (hunk 1 above)
Comment (loweisz) [reply packages/contentful--app-scripts/lib/open/open-settings.js]: Yes I think that makes sense 👍

**Review by kdamball — APPROVED** [review]

**Pull request merged**